fontir/src/coords.rs

```diff
@@ -161,6 +167,32 @@ impl NormalizedCoord {
     }
 }
 
+impl<T> Location<T> {
+    pub fn new(positions: BTreeMap<String, T>) -> Location<T> {
+        Location(positions)
+    }
+
+    pub fn on_axis(name: &str, pos: T) -> Location<T> {
+        Location(BTreeMap::from([(name.to_string(), pos)]))
+    }
+}
+
+impl DesignLocation {
+    pub fn to_normalized(&self, axes: &HashMap<&String, &Axis>) -> NormalizedLocation {
```

Comment:
I might just call this `normalize`? idk, not a blocker or anything, and I'm not totally sure why I have this feeling. Maybe because it takes an argument?

Reply:
I think perhaps to_x makes more sense when you consider each of {design, user, normalized} can reasonable be flipped to any other?